bigbluebutton-html5/imports/ui/components/common/modal/simple/component.jsx

```diff
@@ -59,53 +99,42 @@ class ModalSimple extends Component {
       contentLabel,
       headerPosition,
       'data-test': dataTest,
+      children,
       ...otherProps
     } = this.props;
 
-    const closeModal = (onRequestClose || this.handleDismiss);
-
-    const handleRequestClose = (event) => {
-      closeModal();
-
-      if (event) {
-        event.persist();
-
-        if (event.type === 'click') {
-          setTimeout(() => {
-            document.activeElement.blur();
-          }, 0);
-        }
-      }
-    }
-
     return (
       <Styled.SimpleModal
-        id="simpleModal"
+        id={id || 'simpleModal'}
         isOpen={modalisOpen}
         className={className}
-        onRequestClose={handleRequestClose}
+        onRequestClose={this.handleRequestClose}
         contentLabel={title || contentLabel}
         data={{
           test: dataTest ?? null,
         }}
         {...otherProps}
       >
-        <Styled.Header
-          hideBorder={hideBorder}
-          headerPosition={headerPosition}
-          shouldShowCloseButton={shouldShowCloseButton}
-          modalDismissDescription={intl.formatMessage(intlMessages.modalCloseDescription)}
-          closeButtonProps={{
-            label: intl.formatMessage(intlMessages.modalClose),
-            'aria-label': `${intl.formatMessage(intlMessages.modalClose)} ${title || contentLabel}`,
-            onClick: handleRequestClose,
-          }}
-        >
-          {title}
-        </Styled.Header>
-        <Styled.Content>
-          {this.props.children}
-        </Styled.Content>
+        <FocusTrap active={modalisOpen}>
```

Comment:
```suggestion
        <FocusTrap active={modalisOpen} focusTrapOptions={{ initialFocus: false }}>
```
looks like FocusTrap forces focus on the first element by default... it can be changed by adding `initialFocus: false`

https://github.com/bigbluebutton/bigbluebutton/assets/3728706/0060db63-5939-462d-b561-ce2c51312938